packages/gitgraph-js/rollup.config.js

```diff
@@ -1,3 +1,12 @@
-import build from "../../rollup.config";
+import resolve from "rollup-plugin-node-resolve";
+import commonJS from "rollup-plugin-commonjs";
 
-export default build("gitgraph.core");
+export default {
+  input: "lib/index.js",
+  output: {
+    file: "lib/bundle.umd.js",
+    format: "umd",
+    name: "GitgraphJS",
```

Comment:
UMD should be named apparently (recent Rollup complained otherwise). 

That means the lib will be generated in `window.GitgraphJS` for browsers. Not sure about the impact for npm imports.